diff --git a/common/Makefile b/common/Makefile
index 4003270..f0c502f 100755
--- a/common/Makefile
+++ b/common/Makefile
@@ -58,7 +58,7 @@ else
     CFLAGS += -ffloat-store    
   endif
 endif
-CFLAGS +=  -D JM_PSNR -fno-strict-aliasing -fsigned-char -msse2 -mfpmath=sse  $(STATIC)
+CFLAGS +=  -fno-strict-aliasing -fsigned-char -msse2 -mfpmath=sse  $(STATIC)
 #CFLAGS +=  -D JM_PSNR -ffloat-store -fno-strict-aliasing -fsigned-char $(STATIC)
 FLAGS=  $(CFLAGS) -Wall -I$(INCDIR) -D __USE_LARGEFILE64 -D _FILE_OFFSET_BITS=64
 #FLAGS=  -ffloat-store -Wall -I$(INCDIR) -D __USE_LARGEFILE64 -D _FILE_OFFSET_BITS=64
